tasks/ebpf.py

```diff
@@ -160,9 +159,8 @@ def collect_verification_stats(
 
     with open(VERIFIER_STATS) as f:
```

Comment:
Do we need to open with write permissions here?

Reply:
good catch, i "lost" the change while rebasing!